Encounter Me/Pages/Index.razor

```diff
@@ -135,23 +153,55 @@
 
         var authState = await AuthState;
 
-        string difficulty = capturePoint.DefenseLevel.ToTrailStars();
-
         string infoWindowContent;
 
+        string alegiance;
+
+        if (capturePoint.faction == 0)
+        {
+            alegiance = "Neutral";
+        }
+        else
+        {
+            alegiance = ((Factions)capturePoint.faction - 1).ToString();
+        }
+
         if (authState.User.Identity.IsAuthenticated)
         {
+            Guid userId = Guid.Parse(authState.User.FindFirst(ClaimTypes.UserData).Value);
+            currentUser = await userDataService.GetUserDetails(userId);
+
+            string possibleAction;
+
+
+            //could be done cleaner
+            if (capturePoint.faction == 0)
+            {
+                possibleAction = "Capture";
+            }
+            else if ((capturePoint.faction - 1) == (int)currentUser.Faction.Value)
+            {
+                possibleAction = "Reinforce";
+            }
+            else
+            {
+                possibleAction = "Attack";
+            }
```

Comment:
I understand the first part - if trail is not captured, then you may capture it. But what is going on with reinforce or attack?
In which situation trail can be reinforced and in which - attacked?

Reply:
If trail is of the same faction as the user - "reinforce", else - attack.

Reply:
Okey, but then why it is `(capturePoint.faction - 1)`? Where that -1 comes from?

Reply:
Factions enum does not include 'neutral' faction, and I didn't want to mess with that as it would impact the registration form too. Probably should add 'netural' faction's inclusion into issues list.